clarity_ext/result_file.py

```diff
@@ -0,0 +1,14 @@
+class ResultFile:
```

Comment:
The idea is that the class will be larger with time. I think the domain module is starting to get too large, containing stuff that really doesn't belong together.

This was intended as the start of refactoring the domain module to a package, in order to increase maintainability. I'll use this opportunity to do that.